index.html

```diff
@@ -174,11 +193,11 @@ <h2 class="about-me-header">About Me</h2>
             </div>
         </div>
 
-        <hr>
+        <hr class="hr-aboutme">
         
```

Comment:
- [OPTIONAL] You can remove the margin form this `hr` tag and add it on the top and bottom of the `div` with resoucesection class. You can center this section using Justify-content: space-around; and margin: 150px auto; 

before the edit:
![image](https://user-images.githubusercontent.com/99830261/193172394-0310ba80-377d-4924-aabc-e5acf3272eaa.png)

after the edit:
![image](https://user-images.githubusercontent.com/99830261/193171730-24e49921-f8f0-4d2d-a240-bfb3c9adbad6.png)